internal/cli/up.go

```diff
@@ -34,39 +39,43 @@ func (c *upCmd) register() *cobra.Command {
 func (c *upCmd) run(args []string) error {
 	span := opentracing.StartSpan("Up")
 	defer span.Finish()
-	ctx := opentracing.ContextWithSpan(context.Background(), span)
-	proc, err := tiltd_server.RunDaemon(ctx)
+	ctx := logger.WithLogger(
+		opentracing.ContextWithSpan(context.Background(), span),
+		logger.NewLogger(logLevel(), os.Stdout),
+	)
+
+	logOutput("Starting Tilt…")
+
+	tf, err := tiltfile.Load("Tiltfile")
 	if err != nil {
 		return err
 	}
 
-	defer func() {
-		err := proc.Kill()
-		if err != nil {
-			log.Fatalf("failed to shut down daemon: %v", err)
-		}
-	}()
-
-	dCli, err := tiltd_client.NewDaemonClient()
+	serviceName := args[0]
+	protoServices, err := tf.GetServiceConfig(serviceName)
 	if err != nil {
 		return err
 	}
 
-	logOutput("Starting Tilt…")
+	services := make([]model.Service, len(protoServices))
+	for i, s := range protoServices {
+		services[i] = proto.ServiceP2D(s)
+	}
 
-	tf, err := tiltfile.Load("Tiltfile")
+	env, err := k8s.DetectEnv()
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to detect kubernetes: %v", err)
 	}
 
-	serviceName := args[0]
-	services, err := tf.GetServiceConfig(serviceName)
+	serviceManager := service.NewMemoryManager()
```

Comment:
fyi, this all got replaced by auto-generated DI code

Reply:
ha yeah i just found those conflicts, fixing now